src/extract/geoboundaries.py

```diff
@@ -0,0 +1,38 @@
+import os
+import json
+
+from utils.yaml_api import parse_yaml
+from utils.requests_api import get_json 
+from utils.requests_api import download_url 
+
+
+##
+# https://www.geoboundaries.org/#dataTable
+def get_geoboundaries_adm(admLevel=None):
+    config = parse_yaml('config.yml')
```

Comment:
In principle you shouldn't need to read in the configuration file, but have the parameters passed from the snake rule